packages/automerge-repo-network-websocket/src/NodeWSServerAdapter.ts

```diff
@@ -111,6 +120,9 @@ export class NodeWSServerAdapter
         )
         break
       case "leave":
```

Comment:
Originally, this library was very specific about peers creating individual channels via "join"ing them and then chatting on just that channel about a single docId. This was horribly inefficient if you had lots of docIds and so over time that got de-emphasized. We still do peer discovery via a particular channel (named "sync_channel") but beyond that channels are now really just a key on a message used to multiplex out incoming traffic.

All this is to say that "leave" was never really used but at least conceptually more important when you might join or leave channels based on your interest in a particular document.